app/domain/authentication/authn_jwt/jwt_mandatory_claim.rb

```diff
@@ -0,0 +1,15 @@
+module Authentication
+  module AuthnJwt
+
+    # This class instance holds a JWT standard claim which is
+    # mandatory to validate.
+    class JwtMandatoryClaim
+      attr_reader :name, :value
+
+      def initialize(name:, value:)
```

Comment:
what the : does in the end of each variable?

Reply:
look at the usage of the class, it defined the name of the variables when you initialize the class

Reply:
@tzheleznyak when you have more than one parameter it is best to add `:` at the end of the params. this way caller objects are more clear as to which parameter gets which value and the order doesn't matter. it also forces the caller to set values to the params.